### PR TITLE
Add giz initial configs

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/rqas/giz/s01e01.json [hunk 1]
```diff
@@ -0,0 +1,206 @@
+{
+    "SchemeID": "Scheme-de4d4637",
+    "Name": "S01E01",
```
Comment (as2388) [inline code_schemes/rqas/giz/s01e01.json:3]: Should we start adding the project name here too e.g. "GIZ S01E01"? At the moment the only way to tell which project this code scheme is for is by looking at its location on disk.

File: configurations/docker_image_name.txt [hunk 2]
```diff
@@ -1 +1 @@
-ghcr.io/africasvoices/engagement-data-pipeline:debug_duplicate_origin_ids
+ghcr.io/africasvoices/engagement-data-pipeline:v2.2.0
```
Comment (as2388) [inline configurations/docker_image_name.txt:1]: v3.1.0 is available if upgrading isn't painful

File: configurations/giz_pipeline.py [hunk 3]
```diff
@@ -0,0 +1,316 @@
+from core_data_modules.cleaners import swahili
+
+from src.pipeline_configuration_spec import *
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="GIZ",
+    engagement_database=EngagementDatabaseClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-engagement-databases-firebase-credentials-file.json",
+        database_path="engagement_databases/POOL-KENYA-REBUILD"
```
Comment (as2388) [inline configurations/giz_pipeline.py:9]: Can you add a todo explaining why it's to the rebuild and that it should be corrected when possible?

File: configurations/giz_pipeline.py [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+            AnalysisDatasetConfiguration(
+                 engagement_db_datasets=["location"],
+                 dataset_type=DatasetTypes.DEMOGRAPHIC,
+                 raw_dataset="location_raw",
+                 coding_configs=[
+                     CodingConfiguration(
+                         code_scheme=load_code_scheme("demographics/kenya_ward"),
+                         analysis_dataset="kenya_ward",
+                         analysis_location=AnalysisLocations.KENYA_WARD
+
```
Comment (as2388) [inline configurations/giz_pipeline.py:234]:
```suggestion
```

File: configurations/giz_pipeline.py [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+                                (15, 17): "15 to 17",
+                                (18, 35): "18 to 35",
+                                (36, 54): "36 to 54",
+                                (55, 99): "55 to 99"
+                            }
+                        )
+                    ),
+                ],
+            )
+        ],
```
Comment (as2388) [inline configurations/giz_pipeline.py:273]: disabled?

File: configurations/giz_pipeline.py [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+        ws_correct_dataset_code_scheme=load_code_scheme("ws_correct_dataset"),
+    ),
+    rapid_pro_target=RapidProTarget(
```
Comment (as2388) [inline configurations/giz_pipeline.py:276]: Are you planning continuous sync to textit?

**Review by as2388 — APPROVED** [review]
Approved with a q around continuous sync to textit

**Review by DanielMwendwa — APPROVED** [review]

**Pull request merged**